src/main/java/com/powsybl/openloadflow/network/util/ZeroImpedanceFlows.java

```diff
@@ -148,16 +148,19 @@ private PQ getDescendantZeroImpedanceFlow(Map<LfBus, PQ> descendantZeroImpedance
         }
 
         private void updateBranchFlows(LfBranch branch, LfBus bus, PQ pqBus, PQ pqOtherBus) {
-            if (branch.getBus1().equals(bus)) {
+            if (branch.getBus1() != null && branch.getBus1().equals(bus)) {
                 branch.updateFlows(pqBus.p, pqBus.q, pqOtherBus.p, pqOtherBus.q);
             } else {
                 branch.updateFlows(pqOtherBus.p, pqOtherBus.q, pqBus.p, pqBus.q);
             }
         }
 
         private PQ getBranchFlow(LfBranch branch, LfBus bus) {
-            return branch.getBus1().equals(bus) ? new PQ(branch.getP1().eval(), branch.getQ1().eval())
-                : new PQ(branch.getP2().eval(), branch.getQ2().eval());
+            if (branch.getBus1() != null && branch.getBus1().equals(bus)) {
```

Comment:
Better the previous fix. When a line is antenna the opened bus is null and the returned flow at this side is NaN. With the previous fix the returned flow will be 0.0, the right flow.

Reply:
With previous fix, the last `else` is never reached by unit tests. I have tried during one hour and I think that it could not be reached as it is an adjacent branch. So I think that the right fix is this one.

Reply:
ok